### DISCUSSION TITLE
LineNr highlight is too dark compared to comments

### DISCUSSION THREAD

**Discussion by ghost** [arcticicestudio/nord#234] [Help & Support]
Currently comments are set as such

```
call s:hi("Comment", s:nord3_gui_bright, "", s:nord3_term, "", s:italicize_comments, "")
```

However line numbers are set:

```
call s:hi("LineNr", s:nord3_gui, "NONE", s:nord3_term, "NONE", "", "")
```

s:nord3_term is actually quite difficult to see (as indicated in nordtheme/vim#227) where the solution was to `:set termguicolors`.

The same logic unfortunately seems to apply here too - the darker variant (term3) is used for line numbers making them really difficult to see even with `termguicolors`.

Would you accept a PR to change to:

```
call s:hi("LineNr", s:nord3_gui_bright, "NONE", s:nord3_term, "NONE", "", "")
```

EDIT: I suppose this isn't entirely consistent since we have other areas (folds, etc) which are also using nord3. I don't completely understand the rationale behind using nord3 vs nord3_bright.

I'm not sure what the solution here is. I can override this locally in my vimrc I suppose..

**Comment by svengreb** [comment]
Hi @yiyang-dev :wave:, thanks for your contribution :+1:

The rationale behind `nord3_bright` is documented in https://github.com/arcticicestudio/nord/issues/94 that also includes information why only comments are affected. When `nord3` is really that hard to see I can also recommend to make sure that your system uses the correct color profile for your monitor or maybe use one that is explicitly optimized for text. I also use a `sRGB` color profile (IEC61966-2.1) on my macOS system for my Samsung wide-screen monitor to make sure that colors are "real" (color codes match the rendered color) for best experience for design tasks and also the smoothest and "crispest" rendering for text. `nord3` is good as it is and I often have to make sure that users understand the fact that a good local setup matters. 

Changing the highlighting of line numbers for all users will cause more problems as it solves, but you can still simply resolve it on your own by [customizing the Nord styles to match your own preferences][2] without having to change the source code at all.

[1]: https://en.wikipedia.org/wiki/SRGB
[2]: https://www.nordtheme.com/docs/ports/vim/customization

**Comment by svengreb** [comment]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
